### ISSUE TITLE
Very strange behaviour with pylint and pylintrc file.

### ISSUE THREAD

**Issue by 1oglop1** [PyCQA/prospector#227]
My current configuration:
`.prospector.yaml`

```
pep257:
  disable:
    - D203
    - D212
  enable:
    - D213

pep8:
  enable:
    - W601
  options:
    max-line-length: 121

pylint:
  disable:
    - line-too-long  # Already checked by pep8

ignore-paths:
  - _CI
  - build
```

**What is happening:**
I had `.pylintrc` in the project just because of `line-too-long`. 
And this command `prospector -DFM -s veryhigh` gave me 2 pylint warnings about #ToDo and #Fixme, which was desired.

I have moved `line-too-long` to `.prospector.yaml` (as above)
And removed `.pylintrc`.
Then I did not get warnings from pylint.

After putting empty `.pylintrc` back. It worked again

**Comment by Dunedan** [comment]
I can confirm a different behavior whether an empty `.pylintrc` is available or not. In my specific case I have 0 warnings without an empty `.pylintrc`, but 849 with it.

**Comment by ltalirz** [comment]
prospector has a number of default settings for pylint (to see which, do `prospector --show-profile myfile.py`).
When you provide a `.pylintrc` file, (I believe) the prospector defaults are ignored.

**Comment by Kilo59** [comment]
@ltalirz but custom prospector profile pylint options shouldn't be ignored when there is a `.pylintrc` file correct?

**Comment by carlio** [comment]
In the case of line-too-long from pep8 and pylint, that should go into https://github.com/PyCQA/prospector/blob/master/prospector/blender_combinations.yaml which is for squashing two similar warnings from two tools into one message.

The flow is 'pylint <- configured by .pylintrc if present <- configured by prospector profiles' so profiles take precedent, and by default there will always be at least some strictness profile...

**Comment by carlio** [comment]
This should be fixed now as of pylint 1.7.4
